java/client/src/test/java/glide/api/RedisClientCreateTest.java

```diff
@@ -37,8 +41,9 @@ public void init() {
         channelHandler = mock(ChannelHandler.class);
         commandManager = mock(CommandManager.class);
         connectionManager = mock(ConnectionManager.class);
+        threadPoolResource = mock(ThreadPoolResource.class);
```

Comment:
why do you need to mock this?

Reply:
There's the possibility that the default ELG is not supported on the system (ie Windows) and we want these tests to pass everywhere.

Reply:
if you can't have a default ELG on a system, won't all test cases that involve the default ELG fail - that is, all UTs that use the client without a non-default ELG?
How are you planning on making those tests pass?

Reply:
We have made it so that our unit tests are agnostic to Platform Specific Code. These unit tests for example should even pass on windows even though there is no current support for it. In our end to end tests, they will fail as there is no possible ELG available in the current support.

Reply:
right, so why bother with *mocked* UTs on platforms which aren't supported? If what you gain by mocking is platform support, why is this important? The only thing you learn from the tests is that the mocks work on Windows.